sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -852,7 +852,7 @@ def peek_messages(
             A list of :class:`~azure.storage.queue.QueueMessage` objects. Note that
             next_visible_on and pop_receipt will not be populated as peek does
             not pop the message and can only retrieve already visible messages.
-        :rtype: list(:class:`~azure.storage.queue.QueueMessage`)
+        :rtype: List[QueueMessage]
```

Comment:
Can you leave this one as-is unless Pylint is complaining about it? The way it was will look better in generated docs.

Reply:
Looks like `pylint` is complaining about `:class:` part. I will just remove that part so the return type doc will look like this:
`:rtype: List[~azure.storage.queue.QueueMessage]`

Reply:
Updating this based on previous discussion, will change it to
`list[~azure.storage.queue.QueueMessage]`
Since `list` is a built-in datatype.